src/tools/pkcs11-tool.c

```diff
@@ -4852,12 +4868,10 @@ derive_ec_key(CK_SESSION_HANDLE session, CK_OBJECT_HANDLE key, CK_MECHANISM_TYPE
 		{CKA_KEY_TYPE, &newkey_type, sizeof(newkey_type)},
 		{CKA_SENSITIVE, &_false, sizeof(_false)},
 		{CKA_EXTRACTABLE, &_true, sizeof(_true)},
-		{CKA_ENCRYPT, &_true, sizeof(_true)},
-		{CKA_DECRYPT, &_true, sizeof(_true)},
 		{CKA_WRAP, &_true, sizeof(_true)},
 		{CKA_UNWRAP, &_true, sizeof(_true)}
```

Comment:
I think ecdsa does can not do wrapping either so these should go away too.

Reply:
PKCS11 3.0 "2.3 Elliptic Curve" defines: CKM_ECDH_AES_KEY_WRAP  But I don't see it anywhere in OpenSC.
Do you still want it removed?

Reply:
Thanks for clarification. Lets keep it.